idaes_examples/notebooks/docs/surrogates/alamo_run.alm

```diff
@@ -20,7 +20,7 @@ maxterms 10 10 10 10 10 10 10 10 10 10 10 10 10
 numlimitbasis 1
 
 TRACE 1
-TRACEFNAME C:\Users\dkgun\src\dangunter\idaes-examples\idaes_examples\notebooks\docs\surrogates\alamo_run.trc
+TRACEFNAME D:\Keylogic\examples_Chenyu\examples\idaes_examples\notebooks\docs\surrogates\alamo_run.trc
```

Comment:
You seem to have included some files you shouldn't have. This is probably due to running tests (which indicates we should fix that too), but for this PR you should not include these in the commit.

Reply:
I am not so sure if the test or build documentation lead to this but I believe I should not include this. But I find I no matter how I modify the this `alamo_run.alm` file, it is not tracked and I cannot commit the modification related to this file. When I check the status, it does not show the modificaitons of this file.

Reply:
You are going about it the wrong way - you should just revert the changes you made through git. @lbianchi-lbl is probably best to help you through that (he is better at git than I am).

Reply:
I find the way to revert this change, it just happen when I run `idaesx build`.

Reply:
I think you also need to remove all the things in the nbcache as well - they should not be part of this PR (as @lbianchi-lbl noted we should have these listed in the `.gitignore` so that they don;t get accidentally included).

Also, you might need to add the new notebooks to an indexing page - @dangunter is the one to talk to about what has to be done there (I could not find any documentation of that).

Reply:
I followed the README-developer.md, and the nbcahe files are created by running `idaesx build`. In the instruction, it is said that "Note that the cache for the documentation will be updated to contain the new output cells, which will modify files in idaes_examples/notebooks/nbcache; these files should also be committed and pushed."

Reply:
In addition to the changes in how caching works from #44, I've updated `.gitignore` so that this ALAMO file is also not tracked by Git.